spec/linter-stylelint-spec.js

```diff
@@ -7,117 +7,136 @@ describe('The stylelint provider for Linter', () => {
 
   beforeEach(() => {
     atom.workspace.destroyActivePaneItem();
+    atom.config.set('linter-stylelint.usePreset', true);
+
     waitsForPromise(() => {
-      atom.packages.activatePackage('linter-stylelint');
```

Comment:
Alright so shall I change to:

```js
atom.packages.activatePackage('linter-stylelint').then(() {
   return atom.packages.activatePackage('language-css');
});
```

Or they do not need to wait for each other explicitly like it was before?